### DISCUSSION TITLE
2006/04/08/%E5%8E%BB%E4%BC%91%E6%96%AF%E6%95%A6%E7%9C%8B%E7%90%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#81] [Announcements]
# 2006/04/08/%E5%8E%BB%E4%BC%91%E6%96%AF%E6%95%A6%E7%9C%8B%E7%90%83

有几个同事是体育爱好者，于是约了我一道去休斯敦看姚明的比赛。从我这里到休斯断不算太远，开车3个小时就到了。

https://blog.qizhen.xyz/2006/04/08/%E5%8E%BB%E4%BC%91%E6%96%AF%E6%95%A6%E7%9C%8B%E7%90%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月25日 13:19 编辑
读了这篇文章更向往美国了，呵呵

回复
Tian
2006年05月25日 15:06 编辑
你们找到市中心的华人街没有啊？那里餐馆很多的

回复
Yan
2006年05月14日 00:26 编辑
 
 
   哈哈~~
   胖了哦!!不容易亚!!
 
 
 

回复
Coral
2006年04月18日 11:23 编辑
真是不厚道！我上次的Houston之旅就那样被你给活生生给废了…现在自己跑去那边潇洒快活！回Austin的路上又遇上暴风骤雨了吧，哈！

回复
Xiaohong
2006年04月15日 21:33 编辑
“我们几个人在车里一遍嫉妒美国人民的生活竟然如此腐败，一边盼望下次飓风快点吹过来，”
－－就你们这些人的心态，怎么能过上那种生活呢？

回复
Xiaohong
2006年04月15日 21:30 编辑
都那么大年纪了，怎么还说自己童心未泯啊，shy
 

回复
yoyo
2006年04月12日 18:30 编辑
姚妈妈的馄饨没吃吗
ps 在国外胖了

回复
志磊
2006年04月11日 17:19 编辑
而且你还能在比较前排的位置看球，羡慕ing~~~~~~~~~

回复
志磊
2006年04月11日 17:14 编辑
老大你太幸福啦，居然能现场看球。嘿嘿。哥们儿什么时候才能混到那地步呀，哈哈

回复
jing
2006年04月10日 15:52 编辑
看现场很过瘾的吧？？
